pubspec.yaml

```diff
@@ -89,3 +101,14 @@ flutter:
   #
   # For details regarding fonts from package dependencies,
   # see https://flutter.dev/custom-fonts/#from-packages
+  fonts:
+    - family: Montserrat
+      fonts:
+        - asset: fonts/Montserrat-Bold.ttf
```

Comment:
Što se u aplikaciji samo bold koristi? Gdje je regular? Primjer:
```
  fonts:
    - family: Raleway
      fonts:
        - asset: fonts/Raleway-Light.ttf
          weight: 400
        - asset: fonts/Raleway-Medium.ttf
          weight: 500
        - asset: fonts/Raleway-Bold.ttf
          weight: 800
    - family: Crimson Pro
      fonts:
        - asset: fonts/CrimsonPro-Light.ttf
          weight: 400
        - asset: fonts/CrimsonPro-Medium.ttf
          weight: 500
        - asset: fonts/CrimsonPro-Bold.ttf
          weight: 800
```
znači morate weightove napisati. 400 jer regular, 500 medium, 600 semibold, 700 bold, 800 black